pkg/api/steve/disallow/disallow.go

```diff
@@ -12,6 +12,7 @@ import (
 var (
 	allowAll = map[string]bool{
 		"podsecurityadmissionconfigurationtemplates": true,
+		"projectroletemplatebindings":                true,
```

Comment:
The task on the issue specifies I should make PRTBs available for basic CRUD operations via Steve. But if it's indeed TBD, then I understand. Can remove this part for now...

Reply:
That is part 2 of the issue - I would only do this after part 1 is complete (and I would say passes QA, but that's probably up for debate).

Reply:
This should go in before validation is moved because QA would like to use Steve when testing the validation part

Reply:
> This should go in before validation is moved because QA would like to use Steve when testing the validation part

Not understanding this part, how could QA test validation before validation is moved?

I think if the validation is completed in webhook then this change needs to update rancher-webhook-version so that operations in steve are guaranteed to go through the webhook.

Reply:
Sorry, my point was trying to address the fact that QA needs both this PR and https://github.com/rancher/webhook/pull/228 for validation since they are a part of the same issue.

Reply:
Right, but if this is merged right now then there is nothing tying it to that webhook PR, and updates could go through steve without validation because the old version of webhook is still being used. We should either tag the webhook and include that bump in this PR, or leave this disallow.go change out and wait to add it back until webhook is tagged.